src/pages/PhoneComponent.js

```diff
@@ -0,0 +1,39 @@
+const styles = {
+  container: {
+    backgroundColor: 'white',
+    width: '447px', 
+    height: '778px',
+    borderRadius: '33px',
+    border: '13px solid #BAA7FF',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    backgroundSize: 'cover',
+    backgroundRepeat: 'no-repeat',
+    backgroundPosition: 'center'
+  },
+  notch: {
+    width: '117px',
+    height: '33px',
+    borderRadius: '63px',
+    backgroundColor: '#BAA7FF',
+    marginTop: '12px'
+  },
+  rectangle: {
+    width: '211px',
+    height: '9px',
+    borderRadius: '63px',
+    backgroundColor: '#BAA7FF',
+    marginBottom: '12px'
+  }
+}
+export default function PhoneComponent({children}) {
+  return (
+    <div className="container" style={Object.assign(styles.container, {backgroundImage: `url(${children})`})}>
```

Comment:
I think rather than taking a url as the child for this component, we can have it be more generalized so that users of this component can use it for images in the repo as well.

You can refer to Jacob's component as an example!
https://github.com/IllinoisWCS/IllinoisWCS.github.io/blob/ea950ab997895ff8075aacf5af3ce483854fe041/src/components/computer-window.js#L3-L27